packages/theming/default-theme/src/__tests__/__snapshots__/default-theme.test.ts.snap

```diff
@@ -798,8 +798,8 @@ Object {
     "brandForegroundLinkSelected": "#2899f5",
     "brandStroke1": "#2899f5",
     "brandStroke2": "#004c87",
-    "brandedBackground": "#2b88d8",
-    "brandedBorder": "#deecf9",
+    "brandedBackground": undefined,
+    "brandedBorder": undefined,
     "brandedCheckedBackground": "#484644",
     "brandedCheckedContent": "#faf9f8",
```

Comment:
There's a few of these `undefined` added. Do we know what changed? That seems bad.

Reply:
Yeah I noted this in the PR description and also in my other comment https://github.com/microsoft/fluentui-react-native/pull/2318#issuecomment-1309390185. It's because this package has the outdated iOS color brand ramp. I could get this PR up to 0.31.0, but that would require making more changes that were part of the 0.30.0 update. After this PR is merged I'm planning to immediately start bumping the packages to 0.31.0, at that point there shouldn't ne any undefined values anymore

Reply:
I'm not sure how I feel about our repo being in an intermediate broken state, especially because we publish in every change. Do you have an idea how much extra work the bump to 0.30 and 031 are?

Reply:
The main thing that 0.30.0 has is the corner radius updated ramp, there's about 70 references I would need to update. All 0.31.0 has is the updated color ramp, overall it's not that much extra work. Do you consider this change as introducing a regression?

Reply:
Yes, I would. It would be pretty bad if partners snapped to this version unknowingly, and had a bunch of broken controls in the interim.

Reply:
Talked offline - the newly "undefined" tokens only affect iOS, so this change might not be considered a regression. I am still considering updating this PR to 0.31.0 though

Reply:
Resolved - updated to 0.31.0 (although there is still an open question I have about 0.31.0 for design)